packages/esm-patient-registration-app/src/patient-registration/field/obs/obs-field.component.tsx

```diff
@@ -146,40 +145,32 @@ interface CodedObsFieldProps {
   answerConceptSetUuid?: string;
   label?: string;
   required?: boolean;
+  customConceptAnswers: Array<{ uuid: string; label?: string }>;
 }
 
-function CodedObsField({ concept, answerConceptSetUuid, label, required }: CodedObsFieldProps) {
-  const config = useConfig() as RegistrationConfig;
+function CodedObsField({ concept, answerConceptSetUuid, label, required, customConceptAnswers }: CodedObsFieldProps) {
+  const { t } = useTranslation();
+  const fieldName = `obs.${concept.uuid}`;
+
   const { data: conceptAnswers, isLoading: isLoadingConceptAnswers } = useConceptAnswers(
-    answerConceptSetUuid ?? concept.uuid,
+    customConceptAnswers.length ? '' : answerConceptSetUuid ?? concept.uuid,
   );
 
-  const fieldName = `obs.${concept.uuid}`;
-  const fieldDefinition = config?.fieldDefinitions?.filter((def) => def.type === 'obs' && def.uuid === concept.uuid)[0];
+  const answers = useMemo(
+    () =>
+      customConceptAnswers.length
+        ? customConceptAnswers
+        : isLoadingConceptAnswers
+          ? []
+          : conceptAnswers.map((answer) => ({ ...answer, label: answer.display })),
```

Comment:
Now that I think of it, providing a custom-concept answer shouldn't be allowed.
If someone wants to make any change to the answers, that should be the backend, having a single source of truth, but I think this might become a breaking change.
Thanks!
CC: @ibacher

Reply:
This was added by @makombe in https://github.com/openmrs/openmrs-esm-patient-management/pull/288, reviewed by @donaldkibet . I agree that it is probably bad, but PIH country implementers probably also will need it, as they have great difficulty changing the concepts in their servers (at least as of 2022), CC @mseaton 

Nevertheless this debate should not hold up this PR. But thank you for flagging it @vasharma05

Reply:
I actually broadly think that having a single set of answers for a concept is not always the right thing. What I would prefer is that `conceptConceptAnswers` refers to a custom concept set, rather than having the series of concepts as answers driven purely by frontend configuration. But this isn't really the forum for that discussion.